### PR TITLE
UI Redesign

### PR DESCRIPTION
**Pull request by Snookerballs**
Added Notification, Budget, Stats, and Category Panels for future implementation
Started redesign on GUI
Removed Browser Panel

### CODE REVIEW

File: src/main/java/seedu/address/commons/events/ui/SwapLeftPanelEvent.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package seedu.address.commons.events.ui;
+
+import seedu.address.commons.events.BaseEvent;
+
+/**
```
Comment (JasonChong96) [inline src/main/java/seedu/address/commons/events/ui/SwapLeftPanelEvent.java]: Add author

File: src/main/java/seedu/address/ui/BudgetPanel.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package seedu.address.ui;
+
+import javafx.scene.layout.Region;
+
+/**
```
Comment (JasonChong96) [inline src/main/java/seedu/address/ui/BudgetPanel.java]: Add author

File: src/main/java/seedu/address/ui/CategoriesPanel.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package seedu.address.ui;
+
+import javafx.scene.layout.Region;
```
Comment (JasonChong96) [inline src/main/java/seedu/address/ui/CategoriesPanel.java]: Add author

File: src/main/java/seedu/address/ui/NotificationPanel.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
+package seedu.address.ui;
```
Comment (JasonChong96) [inline src/main/java/seedu/address/ui/NotificationPanel.java]: Add author

File: src/main/java/seedu/address/ui/StatisticPanel.java [hunk 5]
```diff
@@ -0,0 +1,14 @@
+package seedu.address.ui;
+
+import javafx.scene.layout.Region;
```
Comment (JasonChong96) [inline src/main/java/seedu/address/ui/StatisticPanel.java]: Add author

File: src/main/java/seedu/address/ui/Title.java [hunk 6]
```diff
@@ -0,0 +1,16 @@
+package seedu.address.ui;
+
+import javafx.scene.layout.Region;
+
+/**
+ * The Title. Provides the basic application layout containing
```
Comment (JasonChong96) [inline src/main/java/seedu/address/ui/Title.java]: Add author

File: src/main/resources/view/BudgetPanel.fxml [hunk 7]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.StackPane?>
```
Comment (JasonChong96) [inline src/main/resources/view/BudgetPanel.fxml]: Add author

File: src/main/resources/view/CategoriesPanel.fxml [hunk 8]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.StackPane?>
```
Comment (JasonChong96) [inline src/main/resources/view/CategoriesPanel.fxml]: Add author

File: src/main/resources/view/NotificationPanel.fxml [hunk 9]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.ListView?>
+<?import javafx.scene.layout.VBox?>
+
```
Comment (JasonChong96) [inline src/main/resources/view/NotificationPanel.fxml]: Add author

File: src/main/resources/view/StatisticPanel.fxml [hunk 10]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.StackPane?>
+
```
Comment (JasonChong96) [inline src/main/resources/view/StatisticPanel.fxml]: Add author

File: src/main/resources/view/Title.fxml [hunk 11]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.StackPane?>
```
Comment (JasonChong96) [inline src/main/resources/view/Title.fxml]: Add author

File: src/test/java/seedu/address/ui/ExpenseCardTest.java [hunk 12]
```diff
@@ -0,0 +1,72 @@
+package seedu.address.ui;
+
+import static org.junit.Assert.assertEquals;
```
Comment (JasonChong96) [inline src/test/java/seedu/address/ui/ExpenseCardTest.java]: Add author

Comment (JasonChong96) [reply src/test/java/seedu/address/ui/ExpenseCardTest.java]: If this is new

Comment (Snookerballs) [reply src/test/java/seedu/address/ui/ExpenseCardTest.java]: Nope This one is not

**Pull request merged**